content/upcoming/language_migrations.md

```diff
@@ -500,6 +500,13 @@ The thickness of each band corresponds to the value in the dominant eigenvector.
 
 * Regarding **Go**, the popularity stays relatively low, at least in terms of GitHub repositories written in it. However, the dynamic is clearly positive.
 
+## Notebook
+
+To conduct this blog post, I used the following [Jupyter](http://jupyter.org/) notebook
```

Comment:
I used the following Jupyter notebook (nah, no link is needed) to prepare the post: